packages/adapter-vercel/index.d.ts

```diff
@@ -1,4 +1,4 @@
 import { Adapter } from '@sveltejs/kit';
 
-declare function plugin(): Adapter;
+declare function plugin({ external }?: { external?: string[] }): Adapter;
```

Comment:
I wonder if it's worth having the param as `plugin(options?: Options)` instead to simplify the type signature for intellisense, and conforms with the docs wording of "You can pass an `options` argument". But this PR is fine with the current type too.

Reply:
make sense, i made that change